ci/do_circle_ci.sh

```diff
@@ -18,6 +18,8 @@ export NUM_CPUS=8
 # (see https://circleci.com/docs/2.0/executor-types/#using-machine)
 export BAZEL_EXTRA_TEST_OPTIONS="--test_env=ENVOY_IP_TEST_VERSIONS=v4only"
 
+export BAZEL_REMOTE_CACHE="${BAZEL_REMOTE_CACHE:-https://storage.googleapis.com/envoy-circleci-bazel-cache/}"
```

Comment:
Wouldn't it be more appropriate to set this in `.circleci/config.yml`?

Reply:
Yeah, can we make this end-user configurable and document how individual developers can also make use of it? Maybe in https://github.com/envoyproxy/envoy/blob/master/bazel/README.md.

Reply:
(Specifically for when working outside a Docker image)